dbt/include/athena/macros/materializations/models/table/create_table_as.sql

```diff
@@ -47,12 +48,14 @@
   with (
     table_type='{{ table_type }}',
     is_external={%- if table_type == 'iceberg' -%}false{%- else -%}true{%- endif %},
+  {%- if work_group_output_location is none -%}
```

Comment:
as far as I understood this means that if the athena workgroup in use has a default location, the target location will be that. But how will look like the final structure? 
I think that the idea will be to pass work_group_output_location as s3_data_dir in case it set, doing so we keep calling the s3_table_location function.

Reply:
I tried passing in the same `s3_staging_dir` as the default location set in the work group but it didn't seem to work. It does not like you passing in a location when a default location is set on the work group. This is the error I get:
![image](https://user-images.githubusercontent.com/1708162/221258443-c14fae50-9307-4d80-bf81-f2cfe3267752.png)

Reply:
ah right....make sense. We cannot control how the final location will look like as athena will do for us....
Ok what you did make sense, but please be sure that this is documented in the docs:
e.g. in case you use a workgroup with default location, the s3_data_naming does not apply.